codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -227,4 +227,22 @@ update these file paths accordingly.
     }
     fileConsumed.add(entitlementCurrentPath);
   }
+
+  /// Extract entitlements configurations from downloaded zip files.
+  ///
+  /// Parse and store codesign configurations detailed in configuration files.
+  /// File paths of entilement files and non entitlement files will be parsed and stored in [fileWithEntitlements].
+  Future<Set<String>> parseEntitlements(Directory parent, bool entitlements) async {
+    final String entitlementFilePath = entitlements
+        ? fileSystem.path.join(parent.path, 'entitlements.txt')
+        : fileSystem.path.join(parent.path, 'without_entitlements.txt');
+    if (!(await fileSystem.file(entitlementFilePath).exists())) {
+      throw CodesignException('entitilements configuration files are not detected \n'
```

Comment:
Prefer errors that guide the user to what went wrong.

```suggestion
      throw CodesignException('${entitlementFilePath.path} not found'
```

Reply:
Sure, changed.